### ISSUE TITLE
Inaccurate Contrastive Loss in the example?

### ISSUE THREAD

**Issue by gkmeng** [keras-team/tf-keras#469]
Have been following the tutorial below to construct my siamese network. However, I am not sure if the contrastive loss written in this tutorial is correct.

https://keras.io/examples/vision/siamese_contrastive/

In the original paper, similar pair = 0, dissimilar pair = 1, and the eqn of the contrastive loss:

loss = [ (1-y) * D ] + (y * max(margin, D))

However, in the given tutorial, similar pair = 1, dissimilar pair = 0, and the eqn above are still used for calculating the loss. 
Appreciate if anyone can help in clarifying this issue, thank you.
